Stripe/PublicHeaders/STPPaymentIntent.h

```diff
@@ -82,9 +82,16 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  If `status == STPPaymentIntentStatusRequiresSourceAction`, this
- property contains the next action to take for this PaymentIntent.
+ property contains the next source action to take for this PaymentIntent.
+ @deprecated Use nextAction instead
  */
-@property (nonatomic, nullable, readonly) STPPaymentIntentSourceAction* nextSourceAction;
+@property (nonatomic, nullable, readonly) STPPaymentIntentAction* nextSourceAction DEPRECATED_MSG_ATTRIBUTE("Use nextAction instead");
```

Comment:
I think you can/should use `__attribute__((deprecated(...)))` instead, which would provide a fix-it.

I think this is true of every place that `DEPRECATED_MSG_ATTRIBUTE` is used (from AvailabilityMacros.h):

    #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))

AFAIK, we're not interested in cross-compiler support, so might as well always use the fancy 2-argument `clang` attribute: https://clang.llvm.org/docs/AttributeReference.html#deprecated